kinesis.tf

```
@@ -1,41 +1,35 @@
 module "kinesis_label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.3.1"
-  namespace  = "${var.namespace}"
-  name       = "${var.name}"
-  stage      = "${var.stage}"
-  delimiter  = "${var.delimiter}"
-  attributes = "${compact(concat(var.attributes, list("kinesis")))}"
-  tags       = "${var.tags}"
-  enabled    = "${var.enabled}"
+  source  = "cloudposse/label/null"
+  version = "0.24.1"
+
+  context    = module.this.context
+  attributes = compact(concat(module.this.attributes, ["kinesis"]))
```

It is enough just to pass attributes, they will be merged with context attributes
```suggestion
  attributes = ["kinesis"]
```